client/state/data-layer/wpcom/activity-log/rewind/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchRewindStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
+	}, action ) );
+};
+
+const fromApi = response => ( {
+	active: response.use_rewind,
+	firstBackup: response.first_backup_when,
+	error: { message: response.error },
+} );
+
+const updateRewindStatus = ( { dispatch }, { siteId }, next, data ) => {
+	dispatch( {
+		type: REWIND_STATUS_REQUEST,
```

Comment:
In the like post example we sometimes need to fire a follow-up action that's the same (for example, when the response comes back saying it is now different than we requested). With recent changes we need to update the docs but the gist is that this is where we add the data layer bypass, because at that point it's not a data request.

Liking a post happens locally in Calypso. The fact that it synchronizes with WordPress.com is a side-effect that the data layer implements. The actual honest-to-goodness action is "like this post" either way.

In this PR we are looking at something different. We are looking at "I need this rewind data" and the data layer is fulfilling that request. Consider mocking this data in a test or from the developer console. Would we dispatch a request? No, because we already have the data we want to inject. In this case, we would only care about updating the rewind status, thus no request.

`_REQUEST` actions therefore communicate a need to have some data available and this is why the reducers should probably always ignore them - they aren't changing anything in the app. The data might already be available; it might not be; it might be out of date. The data layer takes on the responsibility of fetching that data when it needs to and doing the appropriate thing to get it into state. In this sense we are eliminating `_SUCCESS` and `_FAILURE` actions because they too are almost meaningless to the app, but our reuse is only there for the network requests which we have replaced with new actions.

Thanks for asking and learning!